docs/skopeo-inspect.1.md

```diff
@@ -42,6 +42,7 @@ Use docker daemon host at _host_ (`docker-daemon:` transport only)
 
 Format the output using the given Go template.
 The keys of the returned JSON can be used as the values for the --format flag (see examples below).
+Supports the templating functions available here: https://pkg.go.dev/github.com/containers/common/pkg/report#hdr-Template_Functions
```

Comment:
Soft suggestion: 
```suggestion
Supports the Go templating functions available [here](https://pkg.go.dev/github.com/containers/common/pkg/report#hdr-Template_Functions).
```

Reply:
Done!

Reply:
FWIW using a Markdown link like that looks worse in a man page (`make docs/skopeo-inspect.1 && man docs/skopeo-inspect.1`):
> functions available here ⟨https://pkg.go.dev/github.com/containers/common/pkg/report#hdr-Template_Functions⟩.

So just an “available at https://…” might be a reasonable compromise, at least GitHub’s Markdown rendering turns the URL into a link without extra syntax.

Reply:
@mtrmac Updated per your suggestion